packages/web-components/@sfx/autocomplete/src/autocomplete.ts

```diff
@@ -21,6 +21,10 @@ export default class Autocomplete extends LitElement {
    * The text to use in the header.
    */
   @property({ type: String, reflect: true }) caption: string = '';
+  /**
+   * The optional group name this will check in events.
```

Comment:
This is worded awkwardly. I would word it more like the following:
```suggestion
   * The name of the event group that this component belongs to. This component will dispatch events with this group in their payloads and will react only to events that contain this group.
```
This suggestion applies to all other instances as well.